commit/core.pyx

```diff
@@ -786,7 +786,7 @@ cdef class Evaluation :
             
             self.set_config('confidence_map_filename', confidence_map_filename)
             confidence_map  = nibabel.load( pjoin( self.get_config('DATA_path'), confidence_map_filename) )
-            self.confidence_map_img = confidence_map.get_fdata().astype(np.float64)
+            self.confidence_map_img = np.asanyarray( confidence_map.dataobj ).astype(np.float64)
```

Comment:
Is `float64` necessary? OR `float32` would suffice?

Reply:
This will interact with `y` from `get_y` that is `float64`, I think both have to be the same type